docs/ref/applications.txt

```diff
@@ -147,8 +148,13 @@ configuration::
         # ...
     ]
 
-Again, defining project-specific configuration classes in a submodule called
-``apps`` is a convention, not a requirement.
+This example shows project-specific configuration classes located in a
+submodule called ``apps.py``. This is a convention, not a requirement.
+:class:`AppConfig` subclasses may be defined anywhere.
+
+Where the AppConfig is defined somewhere other than in an ``apps.py``,
+:setting:`INSTALLED_APPS` must contain the dotted path to the configuration
+class.
```

Comment:
I believe this paragraph is misleading. In this example, `anthology` is the "Django project", not a "Django app"; it isn't intended to be in `INSTALLED_APPS`. Therefore, there will be no automatic discovery from `anthology.apps`.

I would keep just the first paragraph here and remove the second one.

Reply:
Or you can revert to my previous attempt:

```
In this situation, :setting:`INSTALLED_APPS` must contain the dotted path to
the configuration class because it lives outside of an application.
```